plugin/evm/testutils/testutils_test.go

```diff
@@ -0,0 +1,81 @@
+package testutils
+
+import (
+	"fmt"
+	"go/parser"
+	"go/token"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// TestMustNotImport fails if a package in plugin/evm imports the testutils package.
+func TestMustNotImport(t *testing.T) {
```

Comment:
I think either a UT like that or a CI script should be sufficient, rather than having a function panicking in prod code.